wrong dependency for sinon-1.10.3.js

Hi all,
After taking the last version from Git, making "npm install" then "npm test", I am getting this error :
# 

Error loading resource file:///C:/.........../eslint-master/eslint-master/node_modules/sinon/pkg/sinon-1.10.3.js (203). Details: Error opening C:/..................../eslint-master/eslint-master/node_modules/sinon/pkg/sinon-1.10.3.js: No such file or directory
TypeError: 'undefined' is not an object (evaluating 'sinon.sandbox')

  file:///C:/............/eslint-master/eslint-master/tests/lib/eslint.js:2040
  file:///C:/................./eslint-master/eslint-master/node_modules/mocha/mocha.js:955
  file:///C:/..................../eslint-master/eslint-master/tests/lib/eslint.js:2082
  file:///C:/....................../eslint-master/eslint-master/node_modules/mocha/mocha.js:955
  file:///C:/........................./eslint-master/eslint-master/tests/lib/eslint.js:2281
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "proxyquire": "^1.0.0",
     "shelljs": "^0.3.0",
     "shelljs-nodecli": "~0.1.0",
-    "sinon": "^1.10.3",
+    "sinon": "~1.10.3",
     "through": "^2.3.6"
   },
   "keywords": [

